### PR TITLE
Tabular module

### PR DESCRIPTION
**Pull request by Sidonivs**
Implementation of the default configuration of the tabular module

### CODE REVIEW

File: s-pipes-core/src/main/java/cz/cvut/spipes/modules/TabularModule.java [hunk 1]
```diff
@@ -104,11 +106,27 @@ ExecutionContext executeSelf() {
...
+            Set<String> columnNames = new HashSet<>();
+            List<RDFNode> columns = new LinkedList<>();
 
-            for (String columnName : header) {
+            for (String columnTitle : header) {
                 Resource columnResource = ResourceFactory.createResource();
-                // TODO make sure normalized names does not clash
-                String columnNameNormalized = normalize(columnName);
+                String columnName = normalize(columnTitle);
+                boolean isDuplicate = !columnNames.add(columnTitle);
```
Comment (blcham) [inline s-pipes-core/src/main/java/cz/cvut/spipes/modules/TabularModule.java:115]: shouldn't be columnNames.add(columnName) ? I think you should find out if isDuplicate is related to normalized name and not the title

**Review by blcham — CHANGES_REQUESTED** [review]
Beside the comment about isDuplicate, it seems fine.

I had little trouble to check as I was expecting some documentation of the module to compare implementation with.

**Review by blcham — CHANGES_REQUESTED** [review]
See my comments about documentation.

File: s-pipes-core/src/main/java/cz/cvut/spipes/modules/TabularModule.java [hunk 2]
```diff
@@ -34,7 +34,27 @@
...
+ * defining the shape of the output RDF data
+ * (i.e. the input metadata values used for the conversion)
+ * using csvw:tableSchema.<br/>
+ * By default, we use the following schema:
+ * <pre><code>
+ * [   a   csvw:Table ;
+ *     csvw:tableSchema
+ *         [   a   csvw:TableSchema ;
+ *             csvw:aboutUrl
+ *                 "http://test-file#row-{_row}"^^csvw:uriTemplate ;
```
[the remark this answers is not in the document]
Comment (blcham) [reply s-pipes-core/src/main/java/cz/cvut/spipes/modules/TabularModule.java:47]: Also as i mentioned in previous comment "Describe how we transform columnName from columnTitle -- if it is not part of the specificaiton"

**Comment by Matthew-Kulich** [comment]
@blcham We don't need this anymore. We should delete it IMO

**Pull request closed without merging**